salt/states/boto_vpc.py

```diff
@@ -433,8 +433,11 @@ def dhcp_options_absent(name=None, dhcp_options_id=None, region=None, key=None,
 
 
 def subnet_present(name, cidr_block, vpc_name=None, vpc_id=None,
-                   availability_zone=None, tags=None, region=None,
-                   key=None, keyid=None, profile=None):
+                   availability_zone=None, tags=None,
+                   route_table_id=None, route_table_name=None,
```

Comment:
Can you move any new kwargs to the end of the list in the function definition? We want to avoid an API change.